scripts/nightly_smoke_ui_tests.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# This job is used for checking compliance frontend health in ephemeral environment
+
+set -ex
+
+export IMAGE="quay.io/cloudservices/compliance-backend"
+export APP_NAME="compliance"  # name of app-sre "application" folder this component lives in
+export COMPONENT_NAME="compliance-backend"  # name of app-sre "resourceTemplate" in deploy.yaml for this component
+export IQE_PLUGINS="compliance"
+export IQE_MARKER_EXPRESSION="ephemeral_ui_nightly"
+export IQE_FILTER_EXPRESSION=""
+export IQE_ENV="ephemeral"
+export IQE_SELENIUM="true"
+export IQE_CJI_TIMEOUT="30m"
+export DEPLOY_TIMEOUT="900"
+export DEPLOY_FRONTENDS="true"
+export COMPONENTS_W_RESOURCES="compliance"
```

Comment:
We're duplicating a lots of constants that we already have in https://github.com/RedHatInsights/compliance-backend/blob/master/Jenkinsfile

Reply:
Unfortunately, by leveraging Bash scripts from the main CICD tooling, one has to set these variables on the environment. 

Being this a Bash script and the PR verification process a Jenkinsfile file, duplication removal comes at the cost of obscurity (you could add these to a variables-only script you source in both the Jenkinsfile and this smoke tests script, but at the cost of having a dependency between them, which is IMHO way worse).